### PR TITLE
chores: remove node-fetch

### PR DESCRIPTION
**Pull request by julien51**
#10838  Description

Now that we use node 18, it has native support for the `fetch` API and we don't need node-fetch anymore!

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: docker/development/eth-node/package.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
     "dependencies": {
         "@nomiclabs/hardhat-ethers": "2.2.1",
         "@unlock-protocol/hardhat-plugin": "0.0.18",
-        "cross-fetch": "3.1.5",
```
Comment (julien51) [inline docker/development/eth-node/package.json:14]: I don't think this was used!

File: packages/networks/src/networks/palm.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    {
+      name: 'Dai',
+      symbol: 'DAI',
+      decimals: 18,
+      address: '0x4C1f6fCBd233241bF2f4D02811E3bF8429BC27B8',
+    },
+  ],
+}
+
+export default palm
```
Comment (searchableguy) [inline packages/networks/src/networks/palm.ts:74]: This does not seem intended to be in this PR.

Comment (julien51) [reply packages/networks/src/networks/palm.ts:74]: Oupsie

**Review by searchableguy — APPROVED** [review]
LGTM. Also let's make sure we use node 18 everywhere to run the code including netlify (we don't use docker image there).

**Comment by julien51** [comment]
Since we now have the node engine in the package file, we should be good correct?

**Pull request merged**